addons/binding/org.openhab.binding.kodi/ESH-INF/i18n/kodi_de.properties

```diff
@@ -53,6 +53,8 @@ channel-type.kodi.input.label = Tastendruck
 channel-type.kodi.input.description = Erm�glicht das Senden eines Eingabebefehls an das Kodi Media Center.
 channel-type.kodi.inputtext.label = Texteingabe
 channel-type.kodi.inputtext.description = Erm�glicht das Senden eines Eingabetextes an das Kodi Media Center.
+channel-type.kodi.inputexecuteaction.label = ExecuteAction
+channel-type.kodi.inputexecuteaction.description = Erm�glicht das Ausf�hren einer Input.Action.
```

Comment:
Omitt using `Input.Action` in the description. Please use a non (detailed) technical translation.

Reply:
no brainer